frame/staking/src/lib.rs

```diff
@@ -3120,12 +3120,13 @@ impl<T> Default for LockStakingStatus<T> {
 	}
 }
 
-impl<T: Trait + Send + Sync> SignedExtension for LockStakingStatus<T> {
+impl<T: Trait + Send + Sync> SignedExtension for LockStakingStatus<T> where
+	<T as Trait>::Call: Dispatchable
```

Comment:
Shouldn't we simply require `Call: Dispatchable` in `frame_system` to avoid repeating that bound?

Reply:
I am not sure. Isn't it missing there for a reason. It seems obvious to have it there. So why it isn't?

Reply:
yeah now that the definition of `Call` in `SignedExtension` is already `Call: Dispatchable`, it makes sense to make the `system::Trait::Call` also have this. Before this PR, call had no bound in either.

Reply:
>  Isn't it missing there for a reason?

Don't quote me on this, but I think no. no bounds were needed so far, so none were added.

Reply:
I added a `Dispatchable` bound which helped to remove this and other few where bounds. However, everyhting that needs to make assumptions about the `Info` or `PostInfo`  tpye still needs extra bounds as those cannot be added to `frame_system::trait::Call`.